test/clj_deconstruct_sigs/data/cosmic_test.cljc

```diff
@@ -0,0 +1,237 @@
+(ns clj-deconstruct-sigs.data.cosmic-test)
```

Comment:
Could you clarify/ how to generate these values?
I think we need a way to trace it back to the original deconstructSigs package for comparison

Reply:
Thanks for the comment! 😸
I have added a function generating the values to `clj-deconstruct-sigs.dev.data` in a4e4022

https://github.com/chrovis/clj-deconstruct-sigs/blob/a4e40229230e03d48c65dd28e8203e55d56e193c/dev/clj_deconstruct_sigs/dev/data.clj#L46-L82